src/events/EventsSection.tsx

```diff
@@ -122,60 +230,269 @@ export const EventsSection = () => {
+                              "eventTypes",
+                              selectedEvents.filter((item) => item !== option)
+                            );
+                            setIsDirty(true);
+                          } else {
+                            setSelectedEvents([...selectedEvents, option]);
+                            setValue("eventTypes", [...selectedEvents, option]);
+                            setIsDirty(true);
+                          }
+                        }}
+                        onClear={clearSelection}
+                        isOpen={selectOpen}
+                        aria-labelledby={"eventType"}
+                        chipGroupComponent={chipGroupComponent()}
+                      >
+                        {events?.enabledEventTypes?.map((option, index) => (
+                          <SelectOption
+                            key={`eventType-${index}`}
+                            value={option}
+                          />
+                        ))}
+                      </Select>
+                    </FormGroup>
+                    <FormGroup
+                      label={t("client")}
+                      fieldId="kc-client"
+                      className="keycloak__user_events_search__form_label"
+                    >
+                      <TextInput
+                        ref={register()}
+                        type="text"
+                        id="kc-client"
+                        name="client"
+                        data-testid="client-searchField"
+                        onChange={(_val, event) => {
+                          const { value } = event.currentTarget;
+                          setValue("client", value);
+                          setIsDirty(true);
+                        }}
+                        defaultValue={
+                          formValues?.client ? formValues?.client : ""
+                        }
+                      />
+                    </FormGroup>
+                    <FormGroup
+                      label={t("dateFrom")}
+                      fieldId="kc-dateFrom"
+                      className="keycloak__user_events_search__form_label"
+                    >
+                      <TextInput
+                        ref={register()}
+                        type="text"
+                        id="kc-dateFrom"
+                        name="dateFrom"
+                        className="pf-c-form-control pf-m-icon pf-m-calendar"
+                        placeholder="yyyy-MM-dd"
+                        data-testid="dateFrom-searchField"
+                        onChange={(_val, event) => {
+                          const { value } = event.currentTarget;
+                          setValue("dateFrom", value);
+                          setIsDirty(true);
+                        }}
+                        defaultValue={
+                          formValues?.dateFrom ? formValues?.dateFrom : ""
```

Comment:
This can be be shortened.
```suggestion
                          formValues?.dateFrom ?? ""
```

Reply:
I like this shorthand - thanks :)